docs/examples/production/rabbitmq.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+  name: production
+spec:
+  replicas: 3
+  resources:
+    requests:
+      cpu: 2
+      memory: 8Gi
+    limits:
+      cpu: 2
+      memory: 8Gi
+  rabbitmq:
+    additionalConfig: |
+      cluster_partition_handling = ignore
+      vm_memory_high_watermark_paging_ratio = 0.99
+      disk_free_limit.relative = 1.0
+  persistence:
+    storageClassName: ssd
+    storage: "128Gi"
+  override:
+    statefulSet:
+      spec:
+        template:
+          spec:
+            containers: []
+            topologySpreadConstraints:
+            - maxSkew: 1
+              topologyKey: zone
```

Comment:
I know this is an example. However I would still use an actual popular label here, like `topology.kubernetes.io/zone` https://kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints/#topologykubernetesiozone